docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -146,21 +171,21 @@ are provided:
 |=======================================================================
 |Strategy |Supported Shapes |Supported Queries |Multiple Shapes
 
-|`recursive` |<<input-structure, All>> |`INTERSECTS`, `DISJOINT`, `WITHIN`, `CONTAINS` |Yes
+|`recursive`  |<<input-structure, All>> |`INTERSECTS`, `DISJOINT`, `WITHIN`, `CONTAINS` |Yes
 |`term` |<<point, Points>> |`INTERSECTS` |Yes
```

Comment:
we need a new line for `vector`?

Reply:
No. I went ahead and removed `vector` altogether and made BKD indexing the default. No `strategy` needed. Its now been deprecated along with all other PrefixTree specific parameters. Just made more sense.